Translator.cpp

```diff
@@ -0,0 +1,20 @@
+// Triangulator.cpp : This file contains the 'main' function. Program execution begins and ends there.
+//
+
+#include <iostream>
```

Comment:
For including header files convention is to include user-defined header files first and then the standard cpp library headers for readability.